algebra/group_power.lean

```diff
@@ -509,6 +509,22 @@ theorem pow_nonneg {a : α} (H : 0 ≤ a) : ∀ (n : ℕ), 0 ≤ a ^ n
 | 0     := zero_le_one
 | (n+1) := mul_nonneg H (pow_nonneg _)
 
+theorem pow_lt {x y : α} {n : ℕ} (Hxy : x < y) (Hxpos : 0 < x) (Hnpos : 0 < n) : x ^ n < y ^ n :=
+begin
+  rw ←nat.sub_add_cancel Hnpos,
+  induction (n - 1), { simpa only [pow_one] },
+  rw [pow_add, pow_add, nat.succ_eq_add_one, pow_one, pow_one],
+  apply mul_lt_mul ih (le_of_lt Hxy) Hxpos (le_of_lt (pow_pos (lt_trans Hxpos Hxy) _))
+end
+
```

Comment:
should be called `pow_lt_pow_of_lt_left`

Reply:
Here again this should hold assuming only that x is nonnegative. There are maybe other instances in this PR where your assumptions can be weakened from positive to nonnegative, please check this.